src/server/getters/get-market-price-history.ts

```diff
@@ -1,31 +1,30 @@
 import * as Knex from "knex";
+import * as _ from "lodash";
 import { Address, TimestampedPrice, MarketPriceHistory } from "../../types";
 import { sortDirection } from "../../utils/sort-direction";
 
-interface MarketPriceHistoryTradesRow {
+interface MarketPriceHistoryTradesRow extends TimestampedPrice {
```

Comment:
Hm, if anything I think the conceptual relationship is the other way -- TimestampedPrice is derived from TradesRow.  That said, I'd sure prefer to avoid inheritance if we can, even if that means we've got some duplicate field names etc.  Does TS have a way to do interface composition?